schematic/store/synapse.py

```diff
@@ -429,6 +436,32 @@ def getProjectManifests(self, projectId: str) -> List[str]:
 
         return manifests
 
+    def upload_project_manifests_to_synapse(self, projectId: str) -> List[str]:
```

Comment:
Sorry if we covered this during the code review, where was this function called again?

Reply:
This function is just one I wrote when I was scoping the project. I kept it in bc it could be useful for people to use, but isn't used in this PR. I can remove it.

Reply:
Ah, I remember now we talked about it! Let's keep it in, but perhaps update the docstring to reflect the functionality? Currently it emphasizes a "get" feature 

> Gets all metadata manifest files across all datasets in a specified project

Reply:
Will do!